Index: Mecanillama.API/Program.cs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Mecanillama.API/Program.cs b/Mecanillama.API/Program.cs
--- a/Mecanillama.API/Program.cs	
+++ b/Mecanillama.API/Program.cs	
@@ -87,7 +87,7 @@
 
 //var connectionString = builder.Configuration.GetConnectionString("DefaultConnection");
 
-var connectionString = builder.Configuration.GetConnectionString("DefaultConnection");
+var connectionString = builder.Configuration.GetConnectionString("AzureDbConnection");
 
 // Database Connection with Standard Level for Information and Errors
 
Index: Mecanillama.API/appsettings.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Mecanillama.API/appsettings.json b/Mecanillama.API/appsettings.json
--- a/Mecanillama.API/appsettings.json	
+++ b/Mecanillama.API/appsettings.json	
@@ -3,7 +3,7 @@
     "Secret": "Place here your secret for token generation"
   },
   "ConnectionStrings": {
-    "DefaultConnection": "server=localhost;user=root;password=luisadrian25;database=mecanillama"
+    "AzureDbConnection": "server=mecanillamadb.mysql.database.azure.com;port=3306;database=mecanillama;user=mecanillama;password=Unapassdificil123"
   },
   "Logging": {
     "LogLevel": {
